docs/e2e-testing.md

```diff
@@ -0,0 +1,121 @@
+## Requirements
+- go 1.16
+- lvm2 packet installed on host machine
+- kubectl/helm
+
+## Build
```

Comment:
These instructions will be outdated soon once we change CI process. Suggest to prepare some script or target in Makefile and use it here and in CI.

Reply:
This comment must be addressed before merging PR.

Reply:
I created an issue for script creating and add TODO in the file